### PR TITLE
vocabularies: harvest latest experiments

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/170

Harvested from greybook using https://gitlab.cern.ch/cds-team/cds-legacy/blob/master/src/wn-cdsweb/bin/grey_book_experiments_synchro#L303

### CODE REVIEW

File: app_data/vocabularies/experiments.yaml [hunk 1]
```diff
@@ -1,3726 +1,7328 @@
...
+  title:
+    en: IS749
+  description: Emission Mossbauer spectroscopy of topological kagome magnets
+  link: https://isolde.cern/active-experiments
+- id: nTOF79
+  title:
+    en: nTOF79
+  description: Measurement of 28,29,30 Si(n,y) capture cross-sections to explain isotopic
+    abundances in presolar grains
+  link: None
```
Comment (anikachurilova) [inline app_data/vocabularies/experiments.yaml:11]: Kept values that were also `None` in the harvested json. Not sure maybe we can just skip them

File: app_data/vocabularies/experiments.yaml [hunk 2]
```diff
@@ -1,3726 +1,7328 @@
...
+    en: nTOF50
+  description: Measurement of the 240 Th (n,f) reaction cross-section at EAR1 and
+    EAR2
+  link: None
+- id: S50
   title:
-    en: ACE AD-4
-- id: ADVANCED VIRGO RE28
+    en: S50
+  description: 'Beta decay of hyperon '
```
Comment (kpsherva) [inline app_data/vocabularies/experiments.yaml:31]:
```suggestion
  description: 'Beta decay of hyperon'
```

File: app_data/vocabularies/experiments.yaml [hunk 3]
```diff
@@ -1,3726 +1,7328 @@
...
+  description: Characterization of the atomic 6D-states in neutral francium using
+    collinear resonance ionization spectroscopy
+  link: None
+- id: UA9
   title:
-    en: AMS RE1
-- id: ANTARES RE6
+    en: UA9
+  description: None
+  link: ' https://ua9.web.cern.ch'
```
Comment (kpsherva) [inline app_data/vocabularies/experiments.yaml:163]:
```suggestion
  link: 'https://ua9.web.cern.ch'
```

File: app_data/vocabularies/experiments.yaml [hunk 4]
```diff
@@ -1,3726 +1,7328 @@
...
+  description: Meas.of Elastic Scatt.in the Coulomb Interfer.Region at the CERN pbarp
+    Collider
+  link: None
+- id: UA7
   title:
-    en: ATLAS
-- id: ATLAS EAGLE
+    en: UA7
+  description: 'Measurement by Silicon Shower Detectors of the Invariant Cross Section
+    of Photons and pi0/s Emitted Close to Zero Degree       '
```
Comment (kpsherva) [inline app_data/vocabularies/experiments.yaml:218]: maybe it makes sense to run some formatter to strip the empty white chars

File: app_data/vocabularies/experiments.yaml [hunk 5]
```diff
@@ -1,3726 +1,7328 @@
...
-- id: IS180
+    en: SATAN
+  description: A Solar Axion Search Using a Decommissioned LHC Test Magnet
+  link: None
+- id: IS621
   title:
-    en: IS180
-- id: IS181
+    en: IS621
+  description: Single-particle behaviour towards the ``island of inversion" - 28,30Mg(d,p)29,31Mg
```
Comment (kpsherva) [inline app_data/vocabularies/experiments.yaml:1113]:
```suggestion
  description: Single-particle behaviour towards the "island of inversion" - 28,30Mg(d,p)29,31Mg
```

**Review by kpsherva — COMMENTED** [review]
I start seeing some syntax issues in the descriptions but I understand they come like this from the source, I think maybe we should decide if we want to clean it on our side or not

File: app_data/vocabularies/experiments.yaml [hunk 6]
```diff
@@ -1,3726 +1,7328 @@
...
-- id: AWAKE
+    en: S45
+  description: $\beta$ parameter of $\Lambda$ decay
+  link: None
+- id: S46
   title:
-    en: AWAKE
-- id: B3
+    en: S46
+  description: Neutral decays of neutral resonances, production of the reaction $\pi^{-}
```
Comment (zzacharo) [inline app_data/vocabularies/experiments.yaml:240]: That is not a string?

File: app_data/vocabularies/experiments.yaml [hunk 7]
```diff
@@ -1,3726 +1,7328 @@
...
+  description: Neutrino Extended Submarine Telescope with Oceanographic Research
+  link: http://www.nestor.org.gr/programme/nestor__scientific_programme.htm
+- id: T211
   title:
-    en: IS21
-- id: IS210
+    en: T211
+  description: K-n Interactions in the c.m.Energy Region around 1 GeV and around 2
+    GeV
+  link: None
```
Comment (zzacharo) [inline app_data/vocabularies/experiments.yaml:1160]: Are these getting propagated into the vocabulary?

**Pull request merged**